DockerfileDB

```diff
@@ -0,0 +1,3 @@
+FROM postgres
+COPY ./sql/create_tables.sql /docker-entrypoint-initdb.d/create_tables.sql
+COPY ./sql/fill_tables.sql /docker-entrypoint-initdb.d/fill_tables.sql
```

Comment:
Could this maybe be done with a `config` map or a "non-empty volume" instead of building another image?

Reply:
Postgres has a weird way it works w init scripts - I looked extensively and also into postgres' init_db scrips. Adding another image or building another shell script to clean the data dir postgres creates is the only working way I have found yet.